python/ray/serve/examples/doc/fastapi/servehandle_fastapi.py

```diff
@@ -6,32 +6,28 @@
 
 app = FastAPI()
```

Comment:
Is it worth keeping this "Scaling up a FastAPI application" section around now that we have the tighter FastAPI integration?

Reply:
Good question. I think it's fine to keep for now but we might consider updating it to use the new "drop-in" support